mod/engine-primitives/pkg/engine-primitives/attributes.go

```diff
@@ -63,18 +63,16 @@ type PayloadAttributes[
 	ParentBeaconBlockRoot common.Root `json:"parentBeaconBlockRoot"`
 }
 
-// NewPayloadAttributes creates a new PayloadAttributes.
-func NewPayloadAttributes[
```

Comment:
this comes from merging https://github.com/berachain/beacon-kit/pull/2068 into this PR